src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -947,5 +949,14 @@ public List<string> StaticFields()
         {
             return typeof(PreferenceSettings).GetMembers(BindingFlags.Static | BindingFlags.NonPublic).OfType<FieldInfo>().Select(field => field.Name).ToList();
         }
+
+        /// <summary>
+        /// Indicates when an instance has been created from a preferences file correctly or not
+        /// </summary>
+        /// <returns></returns>
+        public bool isValidInstance()
```

Comment:
see above suggestion to just use properties.

Reply:
Done.